apps/docs/app/entry.client.tsx

```diff
@@ -1,7 +1,7 @@
 import { CacheProvider } from "@emotion/react";
 import { useState } from "react";
 import { hydrate } from "react-dom";
-import { RemixBrowser } from "remix";
```

Comment:
There is a ton of changes in the `apps/docs/app` folder that's the result of running a migration script. Turns out, Remix wants us to import stuff from different spots